src/editor.c

```diff
@@ -1234,15 +1236,65 @@ editor_get_indent_prefs(GeanyEditor *editor)
 }
 
 
+static inline gint compare_line_indentation(ScintillaObject *sci, gint line1, gint line2)
```

Comment:
FWIW I was told on another project that adding `inline` increases the chances of the compiler inlining it *without* optimisation being turned on, but yeah, it normally doesn't matter.  I'm presuming most distros compile with some level of optimisation, and us devs should use -o2 to get the dataflow analysis needed for some of the warnings.